src/vs/workbench/contrib/testing/browser/testingExplorerView.ts

```diff
@@ -1186,23 +1205,26 @@ interface IActionableElementTemplateData {
 	templateDisposable: IDisposable[];
 }
 
-abstract class ActionableItemTemplateData<T extends TestItemTreeElement> extends Disposable
```

Comment:
Some churn here. Previously we had an abstraction in `ActionableItemTemplateData` when test controllers were a different type of node, but now we represent them as normal tests in our data flows (they're just "hidden" from extensions).